lib/gc/ExecutionEngine/OpenCLRuntime/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ target_compile_options (GcOpenclRuntime PUBLIC -fexceptions -frtti)
 target_include_directories(GcOpenclRuntime PRIVATE
     ${MLIR_INCLUDE_DIRS}
     ${OpenCL_INCLUDE_DIRS}
+    ${PROJECT_SOURCE_DIR}/include
```

Comment:
I think, you can just add GcInterface to target_link_libraries(). It should propagate all the common includes, build options, etc.

Reply:
Yes, that's also a reasonable way.

Reply:
Do you plan to change this?

Reply:
> Do you plan to change this?

Which way do you think is better?

Reply:
Isn't `${PROJECT_SOURCE_DIR}/include` already in include path?

Reply:
The `${PROJECT_SOURCE_DIR}/include` is not in include path, which causes the including error of `gc_utils.h`. But now `gc_utils.h` is moved into `/include/gc`, I will modify this now. Thanks for the reminder!